### ISSUE TITLE
Remove Systers' Logo 

### ISSUE THREAD

**Issue by sakshi1499** [anitab-org/portal#546]
## Description
As a user,
only AnitaB.org logo should be visible.

## Mocks
![image](https://user-images.githubusercontent.com/42520146/83266503-89828500-a1e0-11ea-816c-47d58cd356cb.png)

## Acceptance Criteria
Systers Logo should be removed.
 
## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
1 hour

**Comment by gaurivn** [comment]
I would like to work on this issue.

**Comment by sakshi1499** [comment]
@gaurivn I'll consider this as a first timer issue. Are you currently assigned to any other issue? If not will this be your first time?

**Comment by gaurivn** [comment]
@sakshi1499 , no I am not assigned to any other issue and I am a first timer to portal.

**Comment by sakshi1499** [comment]
Okay cool :) @gaurivn

**Comment by gaurivn** [comment]
Thank you @sakshi1499

**Comment by gaurivn** [comment]
I am working on it, will send updates.

**Comment by gaurivn** [comment]
@sakshi1499 , I am getting this error 
```
createuser: could not connect to database postgres: FATAL:  password authentication failed for user "postgres"
```
Please help me solve this issue. I had tried psql for the password.

**Comment by gaurivn** [comment]
@sakshi1499 , I am getting this error on ubuntu
```
createuser: could not connect to database postgres: FATAL:  password authentication failed for user "postgres"
```
Please help me solve this issue. I had tried psql for the password.

**Comment by SanketDG** [comment]
@gaurivn Did you do a google search first?

https://stackoverflow.com/questions/7695962/postgresql-password-authentication-failed-for-user-postgres

**Comment by sakshi1499** [comment]
@gaurivn You can give the the above link a read. If you are unable to solve the errors, I'll suggest you to run these commands in the given order

`sudo find / -type f -iname pg_hba\.conf`

Then it'll give you the path, run the command below but change the number '11' according to your file path number.

`sudo nano /etc/postgresql/11/main/pg_hba.conf`

Change local all all md5 to peer.

Then start with postgresql commands. This time I guess you'll be able to run it without any error.

**Comment by gaurivn** [comment]
> @gaurivn Did you do a google search first?
> 
> https://stackoverflow.com/questions/7695962/postgresql-password-authentication-failed-for-user-postgres

Yes @SanketDG thank you , I had googled and came across the same link, was trying it out when I started facing technical issues that lasted nearly 3 hrs.

**Comment by gaurivn** [comment]
> @gaurivn You can give the the above link a read. If you are unable to solve the errors, I'll suggest you to run these commands in the given order
> 
> `sudo find / -type f -iname pg_hba\.conf`
> 
> Then it'll give you the path, run the command below but change the number '11' according to your file path number.
> 
> `sudo nano /etc/postgresql/11/main/pg_hba.conf`
> 
> Change local all all md5 to peer.
> 
> Then start with postgresql commands. This time I guess you'll be able to run it without any error.

Thank you very much @sakshi1499 , I tried it but it still didn't work, I'll change local all postgres to peer and try again.

**Comment by sakshi1499** [comment]
Yes change it. It will work don't give up haha @gaurivn

**Comment by gaurivn** [comment]
I am working on it. Got a bit busy with exams....will send a PR soon.

**Comment by sakshi1499** [comment]
@gaurivn No issues :)

**Comment by gaurivn** [comment]
@sakshi1499 , @SanketDG , I could run portal on ubuntu. I would like to suggest few changes to the [ubuntu portal doc](https://docs.google.com/document/d/1nL5c1xxse_ulHbjTgaHHv7XTIf41lQV0msYB7xesJGU/edit).

**Comment by sakshi1499** [comment]
@gaurivn This isn't the right place to suggest it. Suggest the changes on Portal Channel on Zulip under Documentation. :)

**Comment by gaurivn** [comment]
> @gaurivn This isn't the right place to suggest it. Suggest the changes on Portal Channel on Zulip under Documentation. :)

Sure @sakshi1499 , will suggest there.
And Systers and AnitaB.org logo is one image altogether as shown below:
![systersanitablogo](https://user-images.githubusercontent.com/48416306/84744411-44be6280-afc4-11ea-8700-b92e64989683.png)

**Comment by gaurivn** [comment]
How do I proceed? Should I delete this and add a new AnitaB.org logo.
I guess I'll crop it out.

**Comment by emmanuelhashy** [comment]
Hi, sakshi1499 I'd like to work on this issue.
